### DISCUSSION TITLE
Cold start incremental: Cache results of --incremental builds to file system to re-use on subsequent builds

### DISCUSSION THREAD

**Discussion by zachleat** [11ty/eleventy#3992] [Enhancement Queue]
First suggested (to me) here https://github.com/11ty/eleventy/issues/108#issuecomment-513277296

This is the final boss of incremental builds as it would potentially allow a template that never changes to only require a single render step for the entire lifecycle of a project 🤩

**Comment by zachleat** [comment]
> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.

> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by e111077** [comment]
This would be incredibly helpful for integration with https://github.com/google/wireit and its `watch` command.

Wireit watch + eleventy incremental cold start can help prevent the most horrifying string of words in the english language:

`You saved while Eleventy was running, let’s run again. (265 remain)`

**Comment by uplight-dev** [comment]
I'm not sure I understand why this issue is closed.
Does cold start incremental build feature really works now?

I see the documentation page still shows this as TODO.
Also having tried, it will always recompile all md files at every startup of 11ty.

Thank you!

**Comment by agarzola** [comment]
@uplight-dev See first comment on this ticket:

> This repository is now using [lodash style issue management](https://twitter.com/samselikoff/status/991395669016436736) for enhancements. This means enhancement issues will now be closed instead of leaving them open.
> 
> View [the enhancement backlog here](https://github.com/11ty/eleventy/issues?utf8=%E2%9C%93&q=label%3Aneeds-votes+sort%3Areactions-%2B1-desc+). Don’t forget to upvote the top comment with 👍!

**Comment by zachleat** [comment]
Related: https://github.com/11ty/eleventy/issues/3149#issuecomment-2135883987

**Comment by zachleat** [comment]
Related: https://github.com/11ty/eleventy/issues/3324
